docs/building/MlNetMklDeps/mlnetmkl.list

```diff
@@ -4,80 +4,39 @@
 ;---- CBLAS ----
 cblas_sdot
 cblas_sdoti
-
-cblas_sasum
```

Comment:
I assume these are no longer needed?

Reply:
there is no references to them in ML.Net, and they are not on the list of functions needed for upcoming ports. 

---
In reply to: [225262514](https://github.com/dotnet/machinelearning/pull/1261#discussion_r225262514) [](ancestors = 225262514)